tgui/packages/tgui/interfaces/HairPicker.tsx

```diff
@@ -208,26 +215,33 @@ export const HairPickerElement = (props: {
       }}
     >
       <Stack wrap="wrap" height={height}>
-        {hair.map((hair) => (
-          <Stack.Item
-            key={hair.name}
-            className={`Picker${active === hair.icon ? ' Active' : ''}`}
-          >
-            <Tooltip content={hair.name}>
-              <Box
-                position="relative"
-                onClick={() => action({ name: hair.name })}
-              >
-                <DmIcon
-                  icon={icon}
-                  icon_state={`${hair.icon}_s`}
-                  height="64px"
-                  width="64px"
-                />
-              </Box>
-            </Tooltip>
-          </Stack.Item>
-        ))}
+        {hair
+          .filter(
+            (val) =>
+              search.length === 0 || val.name.toLowerCase().includes(search),
+          )
+          .sort((a, b) => (a.name > b.name ? 1 : -1))
+          .map((hair) => (
+            <Stack.Item
+              key={hair.name}
+              className={`Picker${active === hair.icon ? ' Active' : ''}`}
+              height="fit-content"
```

Comment:
```suggestion
              maxHeight="66px"
```
This doesn't seem to make any effect. However you could do instead above. But its not exactly perfect: 
![image](https://github.com/user-attachments/assets/5759dde3-b14d-4896-a430-c4e27a2f6ed2)
![image](https://github.com/user-attachments/assets/3b00cbe8-5ac0-4ebc-b484-653d4fcbca62)
Eliminating the black space just makes it so there's no separate in the normal view e.g. 64px:
![image](https://github.com/user-attachments/assets/609c1f03-c0ed-44cd-bb92-fccdd30a286a)
![image](https://github.com/user-attachments/assets/cc627747-dc6b-43cc-9a07-a415fe12d079)